client/my-sites/drafts/draft-list.jsx

```diff
@@ -152,4 +182,11 @@ var Drafts = React.createClass( {
 	}
 } );
 
-module.exports = DraftList;
+module.exports = connect( ( state, { siteID } ) => {
+	return {
+		isRequestingCounts: isRequestingPostCounts( state, siteID, 'post' ),
+		myCounts: getNormalizedMyPostCounts( state, siteID, 'post' ),
+		everyoneCounts: getNormalizedPostCounts( state, siteID, 'post' ),
+		user: getCurrentUser( state )
```

Comment:
We're only ever using `user.ID`. Should we use the `getCurrentUserId` selector instead? Also might help avoid an error which could occur if `user` was `undefined` when trying to access `user.ID` ([won't happen currently because of user bootstrapping](https://github.com/Automattic/wp-calypso/blob/78439de90bd146c4221184ae00a51760e004e147/client/boot/index.js#L442-L450)).